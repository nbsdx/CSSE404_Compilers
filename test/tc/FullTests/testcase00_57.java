
class Test 
{ //this is a class declaration.
	public static void main(String[] args)
	{			}

}

class Extra extends NextOne
{

}

class NextOne
{
	//oops, can't declare em in this order.
}