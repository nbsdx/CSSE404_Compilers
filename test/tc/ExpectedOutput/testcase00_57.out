Superclass name NextOne not in scope.
